File: ArcherApp/app/src/main/java/me/mxcsyounes/archernotebook/activities/ScoresActivity.java
package me.mxcsyounes.archernotebook.activities;

import android.os.Bundle;
import android.support.design.widget.FloatingActionButton;
import android.support.design.widget.Snackbar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.View;

import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.MobileAds;

import me.mxcsyounes.archernotebook.R;

public class ScoresActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_scores);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        MobileAds.initialize(this, getResources().getString(R.string.app_id_ad_test));

        AdView mAdView = findViewById(R.id.ad_view);

        AdRequest adRequest = new AdRequest.Builder().build();
        mAdView.loadAd(adRequest);

        if (getSupportActionBar() != null)
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        findViewById(R.id.scores_add_score_card).setOnClickListener(v -> {
            // TODO: 18-Aug-18 Launch adding score process
        });

        findViewById(R.id.scores_progress_card).setOnClickListener(v -> {
            // TODO: 18-Aug-18  launch preview of progress
        });

        findViewById(R.id.scores_scores_card).setOnClickListener(v -> {
            // TODO: 18-Aug-18 launch activity of scores
        });
    }

}

File: ArcherApp/app/src/main/java/me/mxcsyounes/archernotebook/activities/ScoresActivity.kt
package me.mxcsyounes.archernotebook.activities;

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.google.android.gms.ads.AdRequest
import com.google.android.gms.ads.MobileAds
import kotlinx.android.synthetic.main.activity_scores.*
import kotlinx.android.synthetic.main.content_scores.*
import me.mxcsyounes.archernotebook.R

class ScoresActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_scores)

        setSupportActionBar(scores_toolbar)

        MobileAds.initialize(this, resources.getString(R.string.app_id_ad_test))

        scores_ad_view.loadAd(AdRequest.Builder().build())

        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        scoresAddScoreCard.setOnClickListener {
            startActivity(Intent(this, AddScoreSheetActivity::class.java))
        }

        scoresProgressCard.setOnClickListener {
            // TODO: 18-Aug-18 Launch progress scores activity
        }

        scoresScoresCard.setOnClickListener {
            startActivity(Intent(this, PreviousScoresActivity::class.java))
        }


    }

}
